package-lock.json

```diff
@@ -3686,12 +3686,12 @@
       }
     },
     "node_modules/@percy/sdk-utils": {
-      "version": "1.16.0",
-      "resolved": "https://registry.npmjs.org/@percy/sdk-utils/-/sdk-utils-1.16.0.tgz",
-      "integrity": "sha512-/nPyK4NCjFGYNVQ7vOivfuEYveOJhA4gWzB7w2PjCkw/Y3kCtu+axRpUiDPEybTz2H6RTvr+I526DbtUYguqVw==",
+      "version": "^1.0.0-beta.63",
```

Comment:
looks like this was a manual edit of the package-lock. the caret before the version number is causing tests to fail